app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -691,7 +684,7 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
             rootView.email.checkEmpty()
 
         var checkBillingInfo = true
-        if (rootView.billingEnabledCheckbox.isChecked) {
+        if (safeArgs.amount > 0) {
```

Comment:
Invalid check. Please check if all tickets are free or donation. Because amount can be 0 due to application of discount code as well

Reply:
So no billing info for donation tickets?

Reply:
Sorry, there will be 2 checks, if all are free/donation and amount == 0, then no billing info

Reply:
Updated